scassandra/src/main/scala/com/evolutiongaming/scassandra/Metadata.scala

```diff
@@ -138,6 +138,8 @@ object KeyspaceMetadata {
       val replication = {
         Sync[F].delay { ToScala.from(keyspaceMetadata.getReplication).toMap }
       }
+
+      val userTypes = ToScala.from(keyspaceMetadata.getUserTypes).toList
```

Comment:
How one is going to access it if it is not exposed in `Metadata` trait?

Reply:
Damn, my fault.

Fixed